plugins/default/lib/nagios.py

```diff
@@ -3,3 +3,12 @@
 WARN=1
 CRIT=2
 UNKNOWN=3
+
+def status_code_to_label(code):
+	return {
+      OK: "OK",
+      WARN: "WARN",
+      CRIT: "CRIT",
+      UNKNOWN: "UNKNOWN"
+    }.get(code, UNKNOWN)
```

Comment:
The default value should probably be the string `"UNKNOWN"`, not a reference to the constant?